corehq/apps/data_interfaces/models.py

```diff
@@ -1115,6 +1116,9 @@ def properties_fit_definition(self, case_properties):
         return all_match or any_match
 
     def when_case_matches(self, case, rule):
+        if CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME in case.case_json:
+            return CaseRuleActionResult()
+
```

Comment:
Nitpicking here, and please feel free to ignore if you think it's not worth it, but what about pushing this out to a function and have something like
```suggestion
if is_copied_case(case):
    return CaseRuleActionResult()
```
The only reason I'm suggesting this change is because it's not immediately obvious what this code is particularly looking for (think of the developer reading this in 1 years time).

Reply:
Sounds good. Addressed in e003e49701807bdb29f89a018ea21d66b13c4ea5